docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -542,6 +542,24 @@ You may have trouble in the configuration if you see Pod statuses like `RunConta
 
 [ubuntu-vagrantfile]: https://github.com/errordeveloper/k8s-playground/blob/22dd39dfc06111235620e6c4404a96ae146f26fd/Vagrantfile#L11)
 
+1. If you see error while run kubectl get po
+
+```
+# kubectl get po                           
+Unable to connect to the server: x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "kubernetes")       
+```
+
+There may be certs not match in your $HOME/.kube, you can run 
```

Comment:
You may want to check if the `$HOME/.kube/config` file contains a valid certificate
and regenerate a certificate if necessary.
Another workaround is to overwrite the default kubeconfig file using the kubeconfig
generated for the "admin" user as shown below: